app/views/spotlight/sir_trevor/blocks/_oembed_block.html.erb

```diff
@@ -1,6 +1,10 @@
-<div class="content-block item-text">
-  <div class="oembed-block <%= "pull-#{ oembed_block.text_align }" if oembed_block.text.present? %>">
+<div class="content-block item-text clearfix">
```

Comment:
I think now that those blocks are wrapped in `col-md`, this should be `row` instead of `clearfix`.